playbook/app/pb_kits/playbook/pb_copy_button/docs/_copy_button_hook_rails.html.erb

```diff
@@ -0,0 +1,16 @@
+<div data-external-copy-from="hookbody">
```

Comment:
Everything looks great! Minor thing: Did you try using the data prop to attach this data attribute to the Body kit itself? If that works, it'll be cleaner code to copy without raw divs!